Source/Node/include/MFRC522Reader.hpp

```diff
@@ -0,0 +1,24 @@
+#include <MFRC522.h>
+
+#include "IReader.hpp"
+
+#define DEFAULT_SS_PIN 10
+#define DEFAULT_RST_PIN 9
+#define DEFAULT_KEY 0xFF
+#define DEFAULT_TRAILER_BLOCK 7
+
+class MFRC522Reader : public IReader
+{
+    private:
```

Comment:
Watch `private:` and `public:` indentation

Reply:
Ditto